frontend/packages/knative-plugin/src/components/overview/__tests__/RevisionsOverviewList.spec.tsx

```diff
@@ -44,6 +48,34 @@ describe('RevisionsOverviewList', () => {
     ).toBe(true);
   });
 
+  const exceedMaxRevisions = [
+    revisionObj,
+    _.set(_.cloneDeep(revisionObj), 'metadata.name', 'overlayimage-tkvz5'),
+    _.set(_.cloneDeep(revisionObj), 'metadata.name', 'overlayimage-bwpxq'),
+    _.set(_.cloneDeep(revisionObj), 'metadata.name', 'overlayimage-n2b7n'),
+    _.set(_.cloneDeep(revisionObj), 'metadata.name', 'overlayimage-m3x8m'),
+  ];
```

Comment:
can add a new describe block and put this in `beforeEach` block as seems used in multiple IT bocks.

Reply:
updated it with data from `traffic-split-utils-mock.ts`